presto-docs/src/main/sphinx/sql/explain.rst

```diff
@@ -139,14 +139,30 @@ IO:
                }
              } ]
            }
-         } ]
+         } ],
+         "estimate" : {
+           "outputRowCount" : 25.0,
+           "outputSizeInBytes" : 2734.0,
```

Comment:
Can you please give an example with higher values? It would be nice to use units here (instead 1000000 use 1M, or 1MB in case of bytes).